src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -150,7 +150,7 @@ def get_lv(self, argument):
             return None
         return api.get_lv(lv_name=lv_name, vg_name=vg_name)
 
-    def setup_device(self, device_type, device_name, tags):
+    def setup_device(self, device_type, device_name, tags, size):
```

Comment:
in bytes or GBs?

Reply:
It's a number string with potentially a multiplier attached to it (k, m, g, ...). Comes form a user argument.